Curve-finance-suite/Unkill/src/agent.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+} from "forta-agent";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools";
+import { provideHandleTransaction, createFinding } from "./agent";
```

Comment:
Wow, importing `createFinding` rather than re-writing another instance in the test file. That's great.